### DISCUSSION TITLE
change display order of author roles

### DISCUSSION THREAD

**Discussion by Zombanana** [gotson/komga#1567] [Ideas]
### Describe your suggested feature

I would need to change the position of "authors".

The standard order of creators is something like that:
- writers
- pencillers
- inkers
- letterers
- editors

Now let assume, I want add a custom field, for example "Artists" and not to use the fields "pencilers" and "inkers".
The order would look like this:
- Writers:
- Colorist:
- Letterers:
- Editors:
- Artists:

Would it be possible to implement manual "ordering" for the custom fields.
This way I could create a order like this:
- Writers:
- Artists:
- Colorists:
- Letterers:
- Editors:

Since the writers and artists are the most important creators of a comic, i don't like them to see on last bottom position :/.
I know, there is a workaround just to create the other roles under the "Artists" field, but that is a hell of work to create all roles manually :/

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to the latest version.
- [X] I will fill out all of the requested information in this form.

**Comment by gotson** [comment]
I don't quite understand, is that in the Edit Book dialog?

**Comment by Zombanana** [comment]
Check this Image:
The "Custom Roles" (in this picture Artists, Colorists, Letterers) are under the "original roles" (pencillers, inkers, colorists, letterers, cover, etc).
![image](https://user-images.githubusercontent.com/97361360/202248999-712ad4cd-b3e1-42e4-8b03-08812a0a5d90.png)

I would be very happy if I could change the position of my custom role "Artists" in this example directly under "writers".
It should look like this:
![image](https://user-images.githubusercontent.com/97361360/202250158-c205e10c-51ca-45a9-a6c7-5e66a9c9046b.png)

I can achieve this by creating the same roles as custom roles and let the "original roles" blank.
But this is kinda dirty workaround.
![image](https://user-images.githubusercontent.com/97361360/202252198-84c24bbe-1932-450e-a571-c149a1f48d90.png)

It would be cool to add some kind of arrows to positions the order of the roles.
![image](https://user-images.githubusercontent.com/97361360/202251219-74b49dcb-0628-433e-8869-7c9c5a3bdb45.png)

Hope this helps better to understand what I mean?^^

**Reply by gotson** [reply]
i honestly don't quite understand why the default doesn't work for you.

You say:
> Since the writers and artists are the most important creators of a comic, i don't like them to see on last bottom position

but **Writer** is the first, and **Penciller** (which is the same as Artist) is the second. They are literally at the top.
